### DISCUSSION TITLE
Ability to switch default gamemode on Android.

### DISCUSSION THREAD

**Discussion by Thepupman1** [ppy/osu#30891] [Q&A]
I couldn't figure out how to switch my default gamemode for so long, and finally, today, I found that in order to switch your default gamemode you had to view your profile page in a web browser and choose desktop site in Chrome's settings in order for the option to change default gamemode to appear. Is there an easier way to do this on android, or can this be added?

**Comment by peppy** [comment]
It's a bit weird and yes, currently missing.

**Comment by Joehuu** [comment]
Relevant web issue: https://github.com/ppy/osu-web/issues/11585
